src/resolvers/githubresolver.ts

```diff
@@ -1,30 +1,47 @@
 import Debug from "debug";
-import * as GitInfo from "hosted-git-info";
 import { SubResolver } from ".";
 
 const debug = Debug("resolverengine:githubresolver");
 
-// hosted-git-info is a godsend, but it doesn't support specific files
-// use a hack to capture third slash a file skipping commitish
-// 1st group - protocol, location, owner, repo
-// 2nd group - file inside repo
-// 3rd group - comittish
-const FILE_LOCATION_REGEX = /^((?:.+:\/\/)?[^:/]+[/:][^/]+[/][^/]+)[/](.+?)(#.+)?$/;
+const BROWSER_LINK_REGEX = /^https?:\/\/github\.com\/([^/]+)\/([^/]+)\/blob\/((?:[^/]+[/])*[^/]+)$/;
```

Comment:
Those regexes are hard to understand, include an explanation for each of those

Reply:
Also they remove support for comittish (`github:user/repo#branch`), gitlab and bitbucket

Reply:
Added into the test excel